webots_ros2_driver/src/plugins/static/Ros2Lidar.cpp

```diff
@@ -19,14 +19,18 @@
 
 #include <webots/robot.h>
 
-namespace webots_ros2_driver {
-  void Ros2Lidar::init(webots_ros2_driver::WebotsNode *node, std::unordered_map<std::string, std::string> &parameters) {
+namespace webots_ros2_driver
+{
+  void Ros2Lidar::init(webots_ros2_driver::WebotsNode *node, std::unordered_map<std::string, std::string> &parameters)
+  {
```

Comment:
That should fix the clang-format issue:
```suggestion
namespace webots_ros2_driver {
  void Ros2Lidar::init(webots_ros2_driver::WebotsNode *node, std::unordered_map<std::string, std::string> &parameters) {
```